extensions/amp-story/0.1/amp-story.js

```diff
@@ -1541,6 +1595,15 @@ export class AmpStory extends AMP.BaseElement {
     const pageToBeInserted = this.getPageById(pageToBeInsertedId);
     const pageToBeInsertedEl = pageToBeInserted.element;
 
+    const allowAutomaticAdInsertion = this.stateService_
```

Comment:
Is the idea with this section to be future proof? If we change the `auto-ads` component this shouldn't be necessary. I could see wanting it in the future if there becomes another way to insert ads, so Ill leave it up to you.

Reply:
Yeah, for future-proofing.